cdap-standalone/src/main/java/co/cask/cdap/StandaloneMain.java

```diff
@@ -232,6 +258,7 @@ public void shutDown() {
         exploreExecutorService.stopAndWait();
       }
       exploreClient.close();
+      metadataService.stopAndWait();
```

Comment:
Since metadata service depends on dataset service and transaction service, I moved the stop of metadata service before dataset service and transaction service were stopped.